java/spring/security/cve/CVE-2016-1000027.yaml

```diff
@@ -0,0 +1,30 @@
+rules:
+- id: CVE-2016-1000027
+  message : |
+    Search for the CVE-2016-1000027
+  patterns :
+    - pattern-either :
+      - pattern : |              
+            HttpInvokerServiceExporter $EXPORTER = new HttpInvokerServiceExporter();
```

Comment:
can use `$EXPORTER = new HttpInvokerServiceExporter(...);` here (general variable, ... makes it more general too)